### PR TITLE
699 double clicking a zim opens multiple windows

### LINKED ISSUE #699: Double-clicking a ZIM opens multiple windows
When double-clicking a ZIM in the Finder (ZIM was not in catalog), the Kiwix app opens with multiple windows:
- one with the homepage of said ZIM ; that's what I expect
- Another on the Reading part, listing my local collection

### PR DESCRIPTION
**Pull request by BPerlakiH**
Fixes #699

### CODE REVIEW

File: App/App_macOS.swift [hunk 1]
```diff
@@ -13,8 +13,15 @@ import Defaults
 import CoreKiwix
 
 #if os(macOS)
```
Comment (BPerlakiH) [inline App/App_macOS.swift:15]: As it turned out this is the official way of closing the application when the last window is closed.

File: App/App_macOS.swift [hunk 2]
```diff
@@ -171,12 +178,6 @@ struct RootView: View {
             navigation.currentItem = .reading
             browser.load(url: url)
         }
```
Comment (BPerlakiH) [inline App/App_macOS.swift:180]: This was our, unofficial way to achieve the same

File: Views/ViewModifiers/FileImport.swift [hunk 3]
```diff
@@ -79,7 +79,9 @@ struct OpenFileHandler: ViewModifier {
                 openedZimFileIDs.forEach { fileID in
                     guard let url = ZimFileService.shared.getMainPageURL(zimFileID: fileID) else { return }
                     #if os(macOS)
-                    NSWorkspace.shared.open(url)
```
Comment (BPerlakiH) [inline Views/ViewModifiers/FileImport.swift:82]: This fixes the multiple windows situation.

**Review by kelson42 — APPROVED** [review]
Lgtm

**Pull request merged**